cargo-dylint/src/main.rs

```diff
@@ -116,6 +116,12 @@ struct Dylint {
 
     #[clap(last = true, help = "Arguments for `cargo check`")]
     args: Vec<String>,
+
+    #[clap(long, value_name = "path", help = "Path to file to pipe stderr to.")]
+    pipe_stderr: Option<String>,
+
+    #[clap(long, value_name = "path", help = "Path to file to pipe stdout to.")]
+    pipe_stdout: Option<String>,
```

Comment:
Could we please move `pipe_stderr` and `pipe_stdout` up so that they appear between `packages` and `quiet`?